packages/polymorphic/src/Polymorphic.types.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Based on https://blog.logrocket.com/build-strongly-typed-polymorphic-components-react-typescript/
+ */
+// TODO: Consider restricting HTML elements to only elements that accept children
```

Comment:
you did this right?

Reply:
No, it currently still accepts any HTML/SVG tag

Reply:
lets rm this comment and make a ticket